README.md

```diff
@@ -1,2 +1,221 @@
+- Elytras don't work
+- Enderpearls don't work
+- Nether portals don't work
+- Carpet teleportation doesn't work
+
+Yes that's right, the only way to carry the enemies treasure from their clan hall to yours is by _walking._ Doesn't sound so easy now, right? You'll need to work together with your team to be protected and score for your clan.
+
+Once you capture the enemies treasure, it becomes deactivated and will only activate in the morning. This means you can only capture their treasure once per day.
+
+<img src="images/ActiveAndDeactiveTreasure.png" alt="Active (Golden) and Deactive (Normal) Treasure" style="width:600px;"/>
+
+### Preventing the other clan from capturing your treasure
+
+So what can you do to prevent the other clan from capturing your treasure? Options include keeping the location of your clan hall a tight keep secret, concealing it, and even booby-trapping the hall. Think Indiana Jones style!
+
+If you manage to kill the carrier of your treasure, it will drop on the ground, and right-clicking it will send it back to your base.
+
+### Getting kills
+
+The other aspect of the Milk Wars is killing. If **and only if an enemy player is signed in to their clan** and you are signed in to yours, you can kill them for an extra point for your clan. Being signed in essentially puts a target on your back, so be careful!
+
+
+## For clan leaders
+
+Congratulations on being a clan leader! This means you're responsible and trustworthy enough to help manage the Milk Wars event.
+
+As a clan leader you have access to the `/clan` (or `/cl`) command. This gives you control over the members of your clan, and most importantly, your clan's flag. Let's take a closer look at how you do those things.
+
+### Managing your clan members
+
+As a clan leader you are able to see the members of your clan and even kick members out if they're misbehaving or cheating with the following commands:
+```
+/clan members list
+/clan members kick <player>
+```
+Note that only admins are allowed to join players into a clan.
+
+### Managing your clan treasure
+
+Your most important role as a clan leader is to manage your clan's treasure. the main command you will do this with is:
+```
+/clan treasure sethome
+```
+This sets the respawn point of the treasure and teleports it to you. Your location becomes the home point of the treasure, and should be in _a visible and well-lit area of your clan hall._ Players shouldn't have to break blocks to get to it! As a clan leader it's your responsibility to make your clan's treasure actually accessible to the enemies to make it fun and enjoyable for everyone. 10 layers of obsidian? No! A fun and challenging parkour to get to the treasure? Perfect!
+
+If for some reason the treasure gets destroyed (which should be impossible) or lost, you can run the following command. Note that the treasure automatically respawns after 24 hours back at its home location, so you should rarely (if ever) need to run this command:
```

Comment:
```suggestion
If for some reason the treasure gets destroyed (which should be impossible for survival players) or lost, you can run the following command. Note that the treasure automatically respawns after 24 hours back at its home location, so you should rarely (if ever) need to run this command:
```

Reply:
Too much info

Reply:
yes, but it is really easy for a creative player to accidentally destroy one.